docs/build.sh

```diff
@@ -18,38 +18,74 @@
 # specific language governing permissions and limitations
 # under the License.
 
-set -e
+set -euo pipefail
 
-FWDIR="$(cd "`dirname "$0"`"; pwd)"
-cd "$FWDIR"
+MY_DIR="$(cd "`dirname "$0"`"; pwd)"
+pushd "${MY_DIR}" || exit 1
 
-NUM_INCORRECT_USE_LITERALINCLUDE_DIRECTIVE=$(grep -inR --include \*.rst 'literalinclude::.\+example_dags' .\
-    tee /dev/tty |\
+if [[ ${APT_DEPS_IMAGE:=""} != "" ]]; then
+    # We are inside the container which means that we should fix permissions of the _build folder files
+    # Those files are mounted from the host!
+    echo "Changing ownership of docs/_build folder to ${AIRFLOW_USER}:${AIRFLOW_USER}"
+    sudo chown ${AIRFLOW_USER}:${AIRFLOW_USER} _build
+    echo "Changed ownership of docs/_build folder to ${AIRFLOW_USER}:${AIRFLOW_USER}"
```

Comment:
Not a problem really, just a pet peeve: Having 2 echo lines for every command that gets executed (`Changing`/ `Changed`,  `Removing`/`Removed`) is probably unnecessary.

Reply:
I see your point. I actually like to have before/after clearly displayed in the logs in case something fails. It's not a big overhead (and of course you could just could just echo when you start. But being a little more verbose is I think better (and you see exactly when things fail - you see it from the logs rather than by looking into the sources of the script). I found it really useful at a number of times. 

Besides in come CI systems (we are considering switching now) you can see the timings for each line - and it makes it a bit easier to see start/stop and calculate length of a task.

Reply:
I hope you are ok with that @gerardo  - it's a diagnostic stuff only. And it's a nit. We can always optimise it cleanup after some teething problems are solved (hopefully not many). I prefer to have a bit more verbose diagnostics first to be able to solve early problems of other people and then to make it smaller after it stabilizes.